src/main/resources/mcmod.info

```diff
@@ -2,14 +2,14 @@
 {
   "modid": "minecolonies",
   "name": "MineColonies",
-  "description": " MineColonies is a Colony/Town Simulator that adds many structures and NPC workers and guards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! BlaBla another spellchecking erroooorrr by Minecolonies and even more qhwefiuhiufhwieufgrhiwefhgiehfiuehfh,iequrdhjfoiqehfriuheifh nedfehfierhfieurhf ewirfhieuhfiherifheif eiuhrfiuehrfiuehfieuhf.",
+  "description": " MineColoneyes is a Coloony/Town Simulater that AIDS many structures and NPC workers and guaaaards that you control and put to work for you! Based around a townhall and village mechanic, there are numerous workers that can be added to your colony, building a busy and vibrant village/town/city within Minecraft. There is no coded limit to the number of NPC's at your disposal, so you can create a small efficient Village dedicated to a particular type of worker, or a sprawling metropolis with many workers. \n It's entirely up to you! BlaBla another spellchecking erroooorrr by Minecolonies and even more qhwefiuhiufhwieufgrhiwefhgiehfiuehfh,iequrdhjfoiqehfriuheifh nedfehfierhfieurhf ewirfhieuhfiherifheif eiuhrfiuehrfiuehfieuhf.",
```

Comment:
### Infos:
- Typo: In word 'NPC's'
- Typo: In word 'nedfehfierhfieurhf'
- Typo: In word 'qhwefiuhiufhwieufgrhiwefhgiehfiuehfh'
- Typo: In word 'erroooorrr'
- Typo: In word 'Minecraft'
- Typo: In word 'Minecolonies'
- Typo: In word 'guaaaards'
- Typo: In word 'townhall'
- Typo: In word 'ewirfhieuhfiherifheif'
- Typo: In word 'Coloneyes'
- Typo: In word 'Coloony'
- Typo: In word 'Simulater'
- Typo: In word 'eiuhrfiuehrfiuehfieuhf'
- Typo: In word 'iequrdhjfoiqehfriuheifh'